### ISSUE TITLE
Reading: Counter-fitting Word Vectors to Linguistic Constraints

### ISSUE THREAD

**Issue by a1da4** [a1da4/paper-survey#211]
## 0. Paper
@inproceedings{mrksic-etal-2016-counter,
    title = "Counter-fitting Word Vectors to Linguistic Constraints",
    author = "Mrk{\v{s}}i{\'c}, Nikola  and
      {\'O} S{\'e}aghdha, Diarmuid  and
      Thomson, Blaise  and
      Ga{\v{s}}i{\'c}, Milica  and
      Rojas-Barahona, Lina M.  and
      Su, Pei-Hao  and
      Vandyke, David  and
      Wen, Tsung-Hsien  and
      Young, Steve",
    booktitle = "Proceedings of the 2016 Conference of the North {A}merican Chapter of the Association for Computational Linguistics: Human Language Technologies",
    month = jun,
    year = "2016",
    address = "San Diego, California",
    publisher = "Association for Computational Linguistics",
    url = "https://aclanthology.org/N16-1018",
    doi = "10.18653/v1/N16-1018",
    pages = "142--148",
}
## 1. What is it?
They propose a new postprocessing approach to consider antonymy.

## 2. What is amazing compared to previous works?
The previous postprocessing (#210) considers only a synonym relation.
That method and the "distributional hypothesis" make antonym words ("east" and "west") similar.
![スクリーンショット 2021-09-29 2 46 26](https://user-images.githubusercontent.com/45454055/135138913-9fa6c9fe-4ed9-46ea-bf93-dec4a7886610.png)
 

## 3. Where is the key to technologies and techniques?
Their method is based on three loss functions.

- synonym attract (#210)
![スクリーンショット 2021-09-29 2 49 23](https://user-images.githubusercontent.com/45454055/135139293-fde0d31e-fabc-4cd4-af3c-0ec1aec8cf25.png)

- antonym rapel
![スクリーンショット 2021-09-29 2 49 32](https://user-images.githubusercontent.com/45454055/135139311-8a369aa5-6e42-4fdc-9fc4-fab459cc79bd.png)

- vector preservation
![スクリーンショット 2021-09-29 2 49 52](https://user-images.githubusercontent.com/45454055/135139362-446b2bab-df39-4822-945e-13d2b2cc7815.png)

Total loss is defined as below:
![スクリーンショット 2021-09-29 2 51 18](https://user-images.githubusercontent.com/45454055/135139575-c815d0b0-bfbd-466e-93bd-02bfa2f9993f.png)

## 4. How did evaluate it?
Word similarity task: their method achieves state-of-the-art performance
![スクリーンショット 2021-09-29 2 52 02](https://user-images.githubusercontent.com/45454055/135139681-6d2bf19a-624e-4979-8711-62fa8038043a.png)

## 5. Is there a discussion?
Benefits: their method can fix relations that the pre-trained model made a mistake in the past
![スクリーンショット 2021-09-29 2 53 13](https://user-images.githubusercontent.com/45454055/135139829-c4b9f663-51af-423d-90cd-10c1e7afc506.png)

**Comment by a1da4** [comment]
#212 
+hypernymy
